src/components/source_editor/BasisText.jsx

```diff
@@ -24,10 +26,13 @@ class BasisText extends React.Component {
 
     // eslint-disable-next-line no-unused-vars
     UNSAFE_componentWillReceiveProps(nextProps, nextContext) {
-        const { content } = this.props;
+        const { content, checks } = this.props;
         if (content !== nextProps.content) {
             this.reformatContentAndUpdateStateIfNoManualEdit(nextProps.content);
         }
+        if (checks !== nextProps.checks) {
+            this.setState({ checks: nextProps.checks });
+        }
```

Comment:
Same as https://github.com/Exabyte-io/cove.js/pull/41/files#r1378311003

Reply:
Got it